src/Helpers/PKCE.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+namespace Auth0\SDK\Helpers;
+
+/**
+ * Class PKCE
+ *
+ * @package Auth0\SDK\Helpers
+ */
+class PKCE
+{
+    /**
+     * Generate a random string of between 43 and 128 characters containing
+     * letters, numbers and "-", ".", "_", "~", as defined in the RFC 7636
+     * specification.
+     *
+     * @param int $length Code verifier length
+     *
+     * @return string
+     *
+     * @see https://tools.ietf.org/html/rfc7636
+     */
+    public static function generateCodeVerifier(int $length = 43): string
```

Comment:
Feel free to extract this method somewhere else or rename this helper class so it can be used for anything else mot necessary related to PKCE.